src/context/GlobalState.tsx

```diff
@@ -1,28 +1,30 @@
+import { getDocs } from "firebase/firestore";
```

Comment:
Directly using `firestore` in components is not recommended. Would suggest creating a service instead (`services/product-service.tsx`) and add functionality there (eg - `getProducts()`).